cmd/jwt/main.go

```diff
@@ -67,14 +67,14 @@ func start() error {
 		return showToken()
 	} else {
 		flag.Usage()
-		return fmt.Errorf("None of the required flags are present.  What do you want me to do?")
+		return fmt.Errorf("none of the required flags are present.  What do you want me to do?")
```

Comment:
Can we consider this a non-breaking change? personally I fully agree with correcting this style but this will break user tests that perform assertions against the explicit errors values. Considering this goes for v4, we can consider it a non-issue but I would think twice if we ever need to backport to v3

Reply:
Since it is in the actual `jwt` application, I would say that this is ok. For the other error changes I would assume that people are trying to compare against the error variables and not the content. But anyway, yes, I would keep this for v4, which promises compatibility but considering it to be a major version bump I would argue this has enough leeway to fix issues like this.